internal/kafka/internal/services/quota/quota_management_list_service.go

```diff
@@ -12,39 +14,57 @@ import (
 	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
 )
 
+const (
+	billingModelStandard = "standard"
+	defaultBillingModel  = billingModelStandard
+)
+
 type QuotaManagementListService struct {
```

Comment:
Given that the behavior of this is not trivial can we add method documentation for the non-trivial methods?